Makefile

```diff
@@ -1,36 +1,53 @@
-default: build start test stop clean
-
-.PHONY: dev
-dev:  ## Build and run containers
-	make build
-	make start
+default: build start create_db populate test stop clean
 
 .PHONY: build
 build:  ## Build containers
-	docker-compose build postgres
-	docker-compose build web
-	docker-compose up -d postgres
-	docker-compose up -d web
-	docker-compose run --rm web /usr/local/bin/python ../create_db.py
-	docker-compose run --rm web /usr/local/bin/python ../test_data.py -p
+	docker-compose build
 
 .PHONY: start
-start:  ## Run containers
+start: build  ## Run containers
 	docker-compose up -d
 
-.PHONY: clean
-clean:  ## Remove containers
-	docker rm openoversight_web_1
-	docker rm openoversight_postgres_1
+.PHONY: create_db
+create_db: start
+	@until docker exec -it openoversight_postgres_1 psql -h localhost -U openoversight -c '\l' postgres &>/dev/null; do \
+		echo "Postgres is unavailable - sleeping..."; \
+		sleep 1; \
+	done
+	@echo "Postgres is up"
+	## Creating database
+	docker-compose run --rm web /usr/local/bin/python ../create_db.py
+
+.PHONY: dev
+dev: build start create_db
```

Comment:
actually - it seems more reasonable to load the test data by default in the dev env, no?

Reply:
Just made the change